src/Core/src/Handlers/WebView/WebViewHandler.Android.cs

```diff
@@ -29,6 +29,11 @@ protected override AWebView CreatePlatformView()
 			platformView.Settings.DomStorageEnabled = true;
 			platformView.Settings.SetSupportMultipleWindows(true);
 
+			if (OperatingSystem.IsAndroidVersionAtLeast(23) && Context?.ApplicationInfo?.Flags.HasFlag(Android.Content.PM.ApplicationInfoFlags.HardwareAccelerated) == false)
```

Comment:
The activities can have different hardware acceleration settings that override the app-level setting, but in out case, I think checking at app level is enough.